HeroVault/HeroVault/Features/Home/HomeScreen/CharacterModel.swift

```diff
@@ -32,7 +32,7 @@ struct Character: Codable, MarvelData {
     let name: String
     let overview: String
     let thumbnail: String
-    let hasBeenfavourited: Bool
+    let hasBeenFavourited: Bool
```

Comment:
```suggestion
    let isFavourited: Bool
```
Avoid using unnecessary words when naming variables